bokehjs/src/lib/models/tools/gestures/tap_tool.ts

```diff
@@ -40,7 +32,16 @@ export class TapToolView extends SelectToolView {
         const did_hit = sm.select(r_views, geometry, final, append)
 
         if (did_hit && callback != null) {
-          cb_data.source = sm.source
+          const frame = this.plot_model.frame
+          const xscale = frame.xscales[renderers[0].x_range_name]
+          const yscale = frame.yscales[renderers[0].y_range_name]
+          const x = xscale.invert(geometry.sx)
+          const y = yscale.invert(geometry.sy)
+          const g = {...geometry, x, y}
+          const cb_data: {
+              geometries: { type: "point", sx: number, sy: number, x: number, y: number }
```

Comment:
This would be more meaningful if the type was `PointGeometry & {x: number, y: number}`.